lib/cep/client.ex

```diff
@@ -55,7 +55,7 @@ defmodule Cep.Client do
 
   defp get_address_from_multiple_sources(cep, sources, error: _, reason: _) do
     source = List.first(sources)
-    client = sources_clients_map()[source]
+    client = sources_clients_map()[source] || source
```

Comment:
I think here there should be a conditional to ensure `source` is a module.  We should have something like this:

```elixir
source = List.first(sources)
client = if is_atom(source) and Code.ensure_loaded?(source)
           source
         else
           sources_clients_map()[source]
         end
```

What do you think?

Reply:
My first implementation was with an `if` like yours, but a coworker suggested de `||` version. 

His arguments:
1 - You are implementing a module to a customizable service, why on earth do you will pass an aleatory module/aton to his configurations?
2 - it's more simple and clean code.

I find valid his arguments and changed to `||` version.

Reply:
I just have an argument against number 1: if you have a simple typo in the code, instead of failing gracefully with something like `source :blabla is not available` it will fail to try to call a function from a module that doesn't exist. This already happens even without the PR though, so I don't mind keeping it, but it's something to be fixed in the future. 👍